### PR TITLE
AMQPC-14: Added mTLS support

### PR DESCRIPTION
**Pull request by nicomz**

### CODE REVIEW

File: amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+= Handle Message Acknowledgment - Mule 4
```
Comment (estebanwasinger) [inline amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc]: is this correct?

File: amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+----
+<amqp:connection host="localhost" port="${rabbitmq.tls.port}" useTls="true" useSasl="true" >
+            <tls:context >
+                <tls:trust-store path="mtls/truststore.jks" password="test" type="jks" />
+                <tls:key-store type="jks" path="mtls/test.jks" alias="test" keyPassword="test" password="test" />
+            </tls:context>
+</amqp:connection>
+----
+`
+== See Also
```
Comment (estebanwasinger) [inline amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc:62]: Wouldl be good to link to this doc https://docs.mulesoft.com/mule-runtime/4.3/tls-configuration

**Review by estebanwasinger — CHANGES_REQUESTED** [review]

**Review by estebanwasinger — DISMISSED** [review]

File: amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc [hunk 3]
```diff
@@ -0,0 +1,65 @@
+= Configure the Mutual TLS Authentication - Mule 4
+:page-aliases: connectors::amqp/amqp-mtls.adoc
+
+The following example shows how to configure Anypoint Connector for AMQP (AMQP Connector) to connect to an AMQP broker with mutual TLS (mTLS) authentication. To configure the global configuration connection you need:
+
+* A client Trust Store that includes the Certificate Authority's certificates required for the server validation.
```
Comment (valkyrie69) [inline amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc]:
```suggestion
* A client truststore that includes the Certificate Authority's certificates that are required for the server validation.
```

File: amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+* A client Key Store that provides the client private key and certificate required for the authentication.
```
Comment (valkyrie69) [inline amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc]:
```suggestion
* A client keystore that provides the client private key and certificate that are required for authentication.
```

File: amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+* The parameters Use TLS and Use SNI must be enabled on the AQMP global configuration in Anypoint Studio.
```
Comment (valkyrie69) [inline amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc]:
```suggestion
* To enable the parameters *Use TLS* and *Use SNI* on the AQMP global configuration in Anypoint Studio.
```

File: amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+. Navigate to the *Global Elements* tab and click *Create*.
+. In the filter box, type `AMQP`.
+. Select *AQMP Config*.
+. Click *OK*.
+. In the *General* tab, for the *Connection* section, set the following parameters:
++
+* *Host*: `localhost`
+* *Port*: `${rabbitmq.tls.port}`
++
+. Select the *Use tls* and *Use sni* parameters box.
```
Comment (valkyrie69) [inline amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc]:
```suggestion
. Select the *Use tls* and *Use sni* parameters boxes.
```

File: amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+. Click the *TLS* tab.
+. For *TLS Configuration* select `Edit inline`.
```
Comment (valkyrie69) [inline amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc]:
```suggestion
. For *TLS Configuration*, select *Edit inline*.
```

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**